src/main/java/games/strategy/engine/framework/ui/background/BackgroundTaskRunner.java

```diff
@@ -45,7 +46,7 @@ private BackgroundTaskRunner() {}
     checkNotNull(backgroundAction);
     checkNotNull(completionAction);
 
-    final WaitDialog waitDialog = new WaitDialog(null, message);
+    final WaitDialog waitDialog = GameRunner.newWaitDialog(message);
```

Comment:
Yup.  The only other code that creates an instance of `WaitDialog` is in `OddsCalculatorPanel`, but it provides a non-`null` parent component.